summoner-cli/test/Test/Show.hs

```diff
@@ -19,5 +19,5 @@ goldenGhcTable =
   , "GHC-8.2.2     base-4.10.1.0   lts-11.22         "
   , "GHC-8.4.4     base-4.11.1.0   lts-12.26         "
   , "GHC-8.6.5     base-4.12.0.0   lts-14.18         "
-  , "GHC-8.8.1     base-4.13.0.0   nightly-2019-12-25"
+  , "GHC-8.8.2     base-4.13.0.0   lts-15.0          "
```

Comment:
Should we remove this padding from all values in the table? Or is it also useful for TUI?

Reply:
This is for the table builder. This pad is supposed to take into consideration the maximum possible alignment, in here I counted the nightly as well. But you are right, I can improve this :)